docroot/themes/custom/mass_theme/templates/block/block--mainnavigation.html.twig

```diff
@@ -31,67 +31,18 @@
  * http://www.w3.org/TR/WCAG-TECHS/H42.html for more information.
  */
 #}
-{% set ariaLabel = heading_id ?? 'main-navigation-heading' %}
-<nav class="ma__header__nav" role="navigation" aria-labelledby="{{ ariaLabel }}" id="main-navigation">
-
-
-  {# Label. If not displayed, we still provide it for screen readers. #}
-  {% if not configuration.label_display %}
```

Comment:
Is this data mapped somewhere else that should be cleaned up since it's no longer used?